WooCommerce/Classes/System/FeatureFlag.swift

```diff
@@ -33,4 +33,11 @@ enum FeatureFlag: Int {
     /// Refunds
     ///
     case refunds
+
+    /// Order List Redesign
+    ///
+    /// This will be removed once the redesign is complete.
+    /// See https://github.com/woocommerce/woocommerce-ios/issues/956.
```

Comment:
Minor suggestion: add the keyword `TODO:`, so when we check what is needed to complete something we don't forget about it.

Reply:
Done in 64c0eeac9b7a743a58ba4e9f3dece40beb7c3caf